src/vm/amd64/unixasmhelpers.S

```diff
@@ -94,7 +94,7 @@ LEAF_END HelperMethodFrameRestoreState, _TEXT
 //  generation will need to altered accordingly.
 //
 // EXTERN_C VOID __stdcall NDirectImportThunk()//
-NESTED_ENTRY NDirectImportThunk, _TEXT
+NESTED_ENTRY NDirectImportThunk, _TEXT, Handler
```

Comment:
To deal with the empty macro arg problem, I would use something more distinct like "NoHandler". Once/if the argument is used by the macro, the distinct name can be special cased.